Assets/Scripts/PlayerAndEnemyContact.cs

```diff
@@ -25,6 +27,7 @@ void OnCollisionEnter2D(Collision2D theCollider)
         {
             Debug.Log("Collided with Pickup");
             theCollider.gameObject.SendMessage("onCollect", this.gameObject);
+            GameObject.Find("ScoreKeeper").SendMessage("Add", CollectionScore);
```

Comment:
if the scene lacks a ScoreKeeper _(for example, given that this feature not ending up on the final alpha)_ , this line is going to throw a null point exception